.codeclimate.yml

```diff
@@ -0,0 +1,63 @@
+plugins:
+  sonar-java:
+    enabled: true
+    config:
+      sonar.java.source: "14"
+
+  pmd:
+    enabled: true
+    checks:
+      AtLeastOneConstructor:
+        enabled: false # Wince we use lombok, it's not actual
+      LawOfDemeter:
+        enabled: false # We love Java Stream API and other chaining-friendly libraries!
+      JUnitTestsShouldIncludeAssert:
+        enabled: false # Spring Boot tests use the underlying statements inside its own
+      BeanMembersShouldSerialize:
+        enabled: false # Most of the declared beans in our app should not be serialized
+      FormalParameterNamingConventions:
+        enabled: false # We think that lambda parameter named as double-underscore is not a crime :)
+    config:
+      rules:
+        - "category/java/bestpractices.xml"
+        - "category/java/codestyle.xml"
+        - "category/java/design.xml"
+        - "category/java/errorprone.xml"
+        - "category/java/multithreading.xml"
+        - "category/java/performance.xml"
+        - "category/java/security.xml"
+
+checks:
```

Comment:
Friends @ACCULA/backend, let's discuss on what thresholds should we set

Reply:
Let's stick with the current numbers and tweak them in the future if needed.

Reply:
I agree with current limits.